vcp-infra/.infra/prod/values.yaml

```diff
@@ -15,6 +15,7 @@ stack:
         - --whitelist-domain=vcp-prod.auth.us-west-2.amazoncognito.com
         - --auth-logging=false
         - --request-logging=false
+        - --set-xauthrequest=true
   services:
```

Comment:
Infra note: enabling `--set-xauthrequest=true` on oauth2-proxy forwards `X-Auth-Request-User`, which the backend now reads to identify the autosave owner. Without this flag the new persistence endpoints would always 401.